k6LoadTests.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+  duration: '10m',
+  vus: 20,
+  //  vus: 1,
+  thresholds: {
+    //avg is around ?800ms? on https://palvelutarjotin-api.test.kuva.hel.ninja
+    http_req_duration: ['p(95)<5000'],
+  },
+};
+
+export default () => {
+  const url = 'https://palvelutarjotin-api.test.kuva.hel.ninja/graphql';
+  const data = 'query=query Places {places(showAllPlaces: true){data{id}}}';
```

Comment:
If the purpose of this is to only "make some load", I would prefer some another query, e.g. organisations, because the place-query actually fetches the data from the LinkedEvents API.

```suggestion
  const data = 'query=query Organisations {organisations {edges {node {id } } } }';
```

Reply:
It's another thing if we really want to test with API dependencies!